### DISCUSSION TITLE
Adding Keywords to multiple photos at once

### DISCUSSION THREAD

**Discussion by catwithbanana** [photoprism/photoprism#2356] [Q&A]
How do I add keywords to multiple photos at once? Let's say I have six photos from one party, and I don't consider it important enough for an album, but I want to give them all a location keyword so I can find them later. How do I perform this as a batch action?

Thanks!

**Comment by graciousgrey** [comment]
Batch edit is not implemented yet, but planned: https://github.com/photoprism/photoprism/issues/271
